buy_together_app/tests/test_sign_up.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from django.template.loader import get_template
+from supplier.models import Supplier
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db()
+class TestSignupPage:
+    def test_signup_template(self):
+        get_template('buy_together_app/sign_up.html')
+
+    def test_signup_supplier(self, client):
+        response = client.post(
+            '/signup/supplier', {'username': "testSignUp",
+                                 'password': "pokemon123",
+                                 'first_name': "ash",
+                                 'last_name': "katchamp",
+                                 'business_name': "pokemon inc"}
+        )
+        assert User.objects.get(username="testSignUp") in list(User.objects.all())
+        assert Supplier.objects.get(
+            supplier_account=User.objects.get(username="testSignUp")).business_name == "pokemon inc"
```

Comment:
What are we asserting here?

Reply:
Try splitting it to multiple statements so it's clear what we're doing, as at the moment it's not that straightforward.

Reply:
Done!